src/figuresClasses.ts

```diff
@@ -2,10 +1,75 @@
+export interface Figure {
+  shape: string;
+  color: string;
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  public shape = 'triangle';
+
+  constructor(
+    public color: string,
+    public a: number,
+    public b: number,
+    public c: number,
+  ) {
+    const sides = [a, b, c].sort((x, y) => x - y);
+
+    if (sides[2] >= sides[0] + sides[1]) {
+      const message =
+        'The sum of any two sides must be greater than the third side.';
+
+      throw new Error(message);
+    }
+  }
+
+  getArea = (): number => {
+    const s = (this.a + this.b + this.c) / 2;
+    const area = Math.sqrt(s * (s - this.a) * (s - this.b) * (s - this.c));
+
+    return Number(area.toFixed(2));
+  };
+}
 
-export class Triangle implements Figure {}
+export class Circle implements Figure {
+  public shape = 'circle';
 
-export class Circle implements Figure {}
+  constructor(
+    public color: string,
+    public radius: number,
+  ) {
+    if (radius <= 0) {
+      throw new Error('Radius must be greater than 0');
+    }
+  }
 
-export class Rectangle implements Figure {}
+  getArea = (): number => {
```

Comment:
[CODE STYLE] - getArea function doesn't need to accept any parameters. All necessary values are already available as object fields. You did it correctly here, but don't forget to follow this rule in all other places.